app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  after_save :update_like_count
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_like_count
```

Comment:
- [ ] [OPTIONAL]
Similarly, let's mark `update_like_count` as _private_ after implementing the `after_save` method.